README.md

```diff
@@ -126,7 +126,7 @@ Please follow the instructions in the [Getting Started](omni/README.md/#getting-
 | Model Category | Model | Type | 
 |----------------------|------------|---------------|
 | **Image Generation** | Qwen-Image, Qwen-Image-Edit | Text-to-Image, Image Editing | 
-| **Image Generation** | Stable Diffusion 3.5 | Text-to-Image, ControlNet | 
+| **Image Generation** | Stable Diffusion 3.5, Z-Image-Turbo | Text-to-Image, ControlNet | 
```

Comment:
Add a new row here. `Z-Image-Turbo` doesn't support ControlNet right now? Only list a series of models in one line.